CONTRIBUTING.md

```diff
@@ -74,5 +74,6 @@ This is your prompt to open a web browser on https://localhost:9876 at which poi
 1. From a fresh checkout of master, create a release branch, ie: `feature/prepare-x.y.z-release`
 2. Update `CHANGELOG.md` by comparing commits to master since the last Github Release
 3. Raise a pull request for your changes, have it reviewed and landed into master.
-4. Switch your local checkout back to the master branch, pull your merged changes and run `./publish-release.sh`.
-5. Create the ARM binaries and attach them to the Github release.
\ No newline at end of file
+4. Make the release on github using the new commit in master.
```

Comment:
The `publish-release.sh` script takes care of creating the github release and attaching the generated binaries to it. It also publishes the client(s) to npm.

Reply:
whew I clearly misunderstood the depth of this script.